docs/src/main/asciidoc/logging.adoc

```diff
@@ -56,13 +56,9 @@ In such cases, you need to use a <<logging-apis,logging adapter>> to ensure that
 
 == Get an application logger
 
-In Quarkus, the most common ways to obtain an application logger are by:
+In Quarkus, the most common ways to obtain an application logger are by <<declaring-a-logger-field,declaring a logger field>>, <<simplified-logging,simplified logging>>, and <<injection-of-a-configured-logger,injecting a configured logger>>.
```

Comment:
To minimize future anchor maintenance (uses the upstream anchor post-fix script) and to support doc skimming: 

```suggestion
In Quarkus, the most common ways to obtain an application logger are by:

* <<declaring-a-logger-field>>
* <<simplified-logging>>
* <<injection-of-a-configured-logger>>
```

Reply:
@michelle-purcell can you explain the issue with the anchors? Does that mean downstream can take "text" when using the << link >> syntax? In this case what text does it use?

Reply:
@cescoffier Hello!
 the `<<declaring-a-logger-field,declaring a logger field>>` approach I am using in the sentence is because I dont want to have Upper capitalization when listing the methods. @michelle-purcell suggests me to return to my previous version in which I had a bullet list containing these approaches. Since they are in a list, they can be naked and their Heading original names will be sourced automatically. 
 
 Yes, downstream limitations doesn't render naked link, thus the name needs to be specified always. However, @gsmet wrote a script that solve this issue, thus Quarkus contributors can use anytime.